.github/workflows/rust.yml

```diff
@@ -138,12 +138,11 @@ jobs:
           toolchain: ${{ matrix.toolchain }}
 
       - name: Install tools
-        shell: bash
-        run: |
-          cargo install cargo-quickinstall
-          cargo quickinstall cargo-make --version 0.36.3 || cargo install --version 0.36.3 cargo-make
-          cargo quickinstall cargo-hack --version 0.5.26 || cargo install --version 0.5.26 cargo-hack
-          cargo quickinstall cargo-nextest --version 0.9.37 || cargo install --version 0.9.37 cargo-nextest
+        uses: taiki-e/install-action@v2
+        env:
+          BINSTALL_LOG_LEVEL: debug
```

Comment:
The debug outputs are pretty large. I'm okay with leaving them in as they are in a separate job, but we can also remove them again, up to you 🙂

Reply:
You are right, removed in [f0072e6](https://github.com/hashintel/hash/pull/2021/commits/f0072e6a59e09ea5256bff5563ebe03eb3c006a1)